examples/llava/clip.cpp

```diff
@@ -1123,19 +1123,22 @@ struct clip_ctx * clip_model_load(const char * fname, const int verbosity = 1) {
                 hparams.image_grid_pinpoints[n] = 0;
         } catch (std::runtime_error & e) {
             hparams.image_grid_pinpoints[0]=0;
+            GGML_UNUSED(e);
         }
```

Comment:
Prefer this pattern instead:

```suggestion
        } catch (std::runtime_error & /*e*/) {
            hparams.image_grid_pinpoints[0]=0;
        }
```

Reply:
Ah that is much nice, I'll update shortly. Thanks!